Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Math/MatrixMxN.ScriptCanvasFunction.xml

```diff
@@ -33,14 +33,24 @@
             <Parameter Name="Source"/>
         </Function>
 
-        <Function Name="MultiplyByVector">
-            <Parameter Name="Source"/>
```

Comment:
is there some kind of deprecation path for these or will graphs that use MultiplyByVector automatically convert to LeftMultiplyByVector?

Reply:
Nope, I just added it to development a few weeks ago, it hasn't hit a release, and there's been no advertisement of the new functionality, so there's almost certainly no dependent code as of yet.